cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/cli/TabReportGenerator.java

```diff
@@ -45,7 +45,7 @@ public final class TabReportGenerator implements FileProvenanceConsumer, AutoClo
           .withNullString("")
           .withRecordSeparator('\n')
           .withHeader(
-              "Last Modified",
+              "Completed Date",
```

Comment:
For backwards compatibility, this should remain Last Modified.

It's the last time that the workflow run was significantly modified. Any additional updates were due to database triggers like a Pinery hash being added, but that doesn't actually affect the workflow run itself.

Reply:
done